Sources/SwiftSyntaxBuilder/ConvenienceInitializers.swift

```diff
@@ -0,0 +1,307 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+@_spi(RawSyntax) import SwiftParser
+@_spi(RawSyntax) import SwiftSyntax
+
+extension BinaryOperatorExpr {
```

Comment:
Personally I like to use `// MARK: - BinaryOperatorExpr` etc to group code as it make it more readable IMO. 
What do you think about that?

Reply:
I don’t have strong opinions because each type as a single `extension` at the moment. Since you prefer them, I added them 😄